### ISSUE TITLE
Get full OCaml backtrace error or more error information in semgrep.live

### ISSUE THREAD

**Issue by aryx** [returntocorp/semgrep#609]
See https://semgrep.live/AWg
Which returns a FatalError.
It would be great if from semgrep.live I had access to more information about the error, such as
the full OCaml backtrace that led to this error, that way I could more quickly diagnose if it's
a problem in the lexer, or parser, or CST -> AST, or AST -> AST generic conversion.

**Comment by aryx** [comment]
Like the AST dump, maybe this should be accessible only in a "debug" mode. We don't want users
to  see the full backtrace.

**Comment by DrewDennison** [comment]
Of course. Great request. I think this will have to be plumb through the python code first cc @brendongo
